mne/annotations.py

```diff
@@ -430,28 +431,76 @@ def _write_annotations(fid, annotations):
     end_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
 
 
-def read_annotations(fname):
-    """Read annotations from a FIF file.
+def read_annotations(fname, sfreq='auto', uint16_codec=None):
+    r"""Read annotations from a file.
+
+    This function reads a .fif, .fif.gz, .vrmk, .edf or .set file and makes an
+    :class:`mne.Annotations` object.
 
     Parameters
     ----------
     fname : str
         The filename.
 
+    sfreq : float | 'auto'
+        The sampling frequency in the file. This parameter is necessary for
+        \*.vmrk files as Annotations are expressed in seconds and \*.vmrk files
+        are in samples. For any other file format, ``sfreq`` is omitted.
+        If set to 'auto' then the ``sfreq`` is taken from the \*.vhdr
+        file that has the same name (without file extension). So data.vrmk
+        looks for sfreq in data.vhdr.
+
```

Comment:
we do not typically have these extra newlines (unless they are necessary for the RST to render; here they are not)